.github/workflows/jax_setup.yml

```diff
@@ -0,0 +1,137 @@
+name: Test for DeepChem Jax
+on:
+  push: # ci work when pushing master branch
+    branches:
+      - master
+  pull_request: # ci work when creating a PR to master branch
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Cache pip modules for Linux
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('env.*.yml') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Build DeepChem
+      run: |
+        python -m pip install --upgrade pip
+        pip install tensorflow'>=2.3,<2.4'
```

Comment:
I think this line is outdated (in the original as well). Can you swap this to `pip install tensorflow==2.4` for now?